docs/source/guides/index.rst

```diff
@@ -18,4 +18,5 @@ and ask in the bentoml-users channel.
   custom_artifact
   custom_input_adapter
   yatai_service
+  aws sagemaker end-to-end workflow  
```

Comment:
Replace spaces with _ underscores -- that way, this will auto-generate a link in the table of contents.